src/table/table.scss

```diff
@@ -337,6 +373,7 @@
     justify-content: center;
     align-items: center;
     flex-grow: 1;
+    width: 100%;
```

Comment:
what about this one? I don't see any difference when I toggle it off

Reply:
Must be a leftover. Removed.